src/OrchardCore/OrchardCore.Users.Core/Authentication/CacheTicketStore.cs

```diff
@@ -17,7 +17,7 @@ public class CacheTicketStore : ITicketStore
 
     private readonly IHttpContextAccessor _httpContextAccessor;
     private IDataProtector _dataProtector;
-    private ILogger<CacheTicketStore> _logger;
+    private ILogger _logger;
 
     public CacheTicketStore(IHttpContextAccessor httpContextAccessor)
```

Comment:
Why the logger is not injected here?

Reply:
Because the service itself is not injected, a new one is built to set a `SessionStore` option

https://github.com/OrchardCMS/OrchardCore/blob/89eafc5a8e4bb03adada0f52114882331d7e00b0/src/OrchardCore.Modules/OrchardCore.Users/CacheTicketStoreStartup.cs#L28-L34

That's why the services including the logger are resolved lazily on demand.